docs/matlab_octave_interface/index.md

```diff
@@ -0,0 +1,101 @@
+# Matlab + Simulink and Octave Interface
+
+In order to use `acados` from Octave or Matlab, you need to create the `acados` shared libraries  using either the `CMake` or `Make` build system, as described [on the installation page](../installation/index.md).
+
+## Getting started
+Check out the examples in [`<acados_root>/examples/acados_matlab_octave/getting_started`](https://github.com/acados/acados/tree/master/examples/acados_matlab_octave/getting_started) to get started with this interface.
+
+The examples require an installation of `CasADi` to generate the model functions.
+The `getting_started` example offers the option to attempt to automatically download the correct version in the recommended folder.
+Detailed instructions for a manual installation can be found in the last section of this page [Setup CasADi](#setup-casadi).
+
+The problem formulation is stated in [this PDF](https://github.com/acados/acados/tree/master/docs/problem_formulation/problem_formulation_ocp_mex.pdf).
+
+### Linux / macOS
+To run the examples, navigate into the folder of the example you want to run and execute the following command:
+```
+source env.sh # Which can be found in the folder of one of the examples
+```
+
+If you want to run an `acados` example from another folder, you need to export the envirionment variable `ACADOS_INSTALL_DIR` properly.
+In the `env.sh` file it is assumed that `ACADOS_INSTALL_DIR` is folders up from the directory, in which the example is located.
```

Comment:
'is folders up from the directory' -> 'is a parent directory of' ...?

Reply:
actually a "two" is missing, so this phrasing doesn't work :sweat_smile: